hmda-regulator/src/main/resources/application.conf

```diff
@@ -3,16 +3,16 @@ akka {
   http.server.preview.enable-http2 = on
   http.server.request-timeout=20800s
   //Dev Schedules (fire every 10 seoncds)
-//  PanelScheduler = """akka.quartz.schedules.PanelScheduler.expression = "0 0/2 * * * ?""""
-//  LarScheduler = """akka.quartz.schedules.LARScheduler.expression = "0 0/2 * * * ?""""
-//  TsScheduler = """akka.quartz.schedules.TsScheduler.expression = "0 0/2 * * * ?""""
+  //  PanelScheduler = """akka.quartz.schedules.PanelScheduler.expression = "0 0/2 * * * ?""""
+  //  LarScheduler = """akka.quartz.schedules.LARScheduler.expression = "0 0/2 * * * ?""""
+  //  TsScheduler = """akka.quartz.schedules.TsScheduler.expression = "0 0/2 * * * ?""""
   //Prod Schedules
   //fire at 11:00PM everyday
-  PanelScheduler = """akka.quartz.schedules.PanelScheduler.expression = "0 20 06 * * ?""""
- // fire at 11:10PM everyday
-  LarScheduler = """akka.quartz.schedules.LARScheduler.expression ="0 55 05 * * ?""""
- // fire at 11:20PM everyday
-  TsScheduler = """akka.quartz.schedules.TsScheduler.expression = "0 10 06 * * ?""""
+  PanelScheduler = "akka.quartz.schedules.PanelScheduler.expression = "${PANEL_SCHEDULE}""
```

Comment:
It might be worthwhile keeping the original schedules as a default.

Reply:
Ill add a note outlining what these values could reflect. Really need to add this to Regulator documentation